satpy/tests/utils.py

```diff
@@ -355,3 +355,117 @@ def __call__(self, dsk, keys, **kwargs):
             raise RuntimeError("Too many dask computations were scheduled: "
                                "{}".format(self.total_computes))
         return dask.get(dsk, keys, **kwargs)
+
+
+def make_a_fake_scene(content_dict, daskify=False, area=True,
```

Comment:
Ok, good that `fake` made its way in the name.
Now, I think `_a` should be removed. I believe shorter function names are easier to type.
Regarding `make`: I personally would remove it, but I'm aware that some standards recommend to have a verb in function names. I think however that `fake_scene` is as clear as `make_fake_scene`.
But this is just an opinion, so I'll let you decide what you want to do.

Reply:
Changed to `make_fake_scene`.